modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/NotificationQueueEntryStatusDataRenderer.tsx

```diff
@@ -12,8 +12,50 @@
  * details.
  */
 
+import ClayPopover from '@clayui/popover';
 import classNames from 'classnames';
-import React from 'react';
+import React, {useState} from 'react';
+
+export function NotificationQueueEntryToDataRenderer({value}: {value: string}) {
+	const [isPopoverVisible, setPopoverVisible] = useState(false);
+
+	const POPOVER_MAX_WIDTH = 256;
+
+	const to = value;
+	const toArray = to.split(',');
+	const lastItem = toArray.length > 1 ? toArray.pop() : null;
+	const toTooltipText =
+		toArray.length > 1
+			? toArray.join(',') +
+			  ' ' +
+			  Liferay.Language.get('and').toLowerCase() +
+			  lastItem +
+			  '.'
+			: to;
+	const toValue = to.length > 25 ? to.slice(0, 25).trim() + '...' : to;
```

Comment:
Hey @alinedoleron, you can pass this ternary condition directly in where you call this const and delete it